<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMindMapDtFileTool</class>
 <widget class="QWidget" name="CMindMapDtFileTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="m_btnAttachFile">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="mindstorm.qrc">
       <normaloff>:/mindstorm/icons/ui-attach.png</normaloff>:/mindstorm/icons/ui-attach.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_btnDeleteAttachedFile">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="mindstorm.qrc">
       <normaloff>:/mindstorm/icons/ui-document-delete.png</normaloff>:/mindstorm/icons/ui-document-delete.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_btnRemoveAttach">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="mindstorm.qrc">
       <normaloff>:/mindstorm/icons/ui-delete.png</normaloff>:/mindstorm/icons/ui-delete.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mindstorm.qrc"/>
  <include location="mindstorm.qrc"/>
 </resources>
 <connections/>
</ui>
